spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/flyway/FlywayAutoConfiguration.java

```diff
@@ -147,9 +164,8 @@ private void checkLocationExists(String... locations) {
 				Assert.state(locations.length != 0,
 						"Migration script locations not configured");
 				boolean exists = hasAtLeastOneLocation(locations);
-				Assert.state(exists,
-						() -> "Cannot find migrations location in: " + Arrays.asList(
-								locations)
+				Assert.state(exists, () -> "Cannot find migrations location in: "
```

Comment:
Revert?

Reply:
Not sure what you mean by that?

Reply:
Unnecessary formatting.

Reply:
That's fine. We deal with formatting issues all the time when we merge things. No need to resubmit the PR just for that.

Reply:
I'm pretty sure this actually changed after I ran the [eclipse formatter](https://github.com/spring-projects/spring-boot/blob/master/eclipse/eclipse-code-formatter.xml) over it.